doc/API.md

```diff
@@ -42,13 +43,14 @@ This command does not take any parameter for now.
 
 #### Response
 
-| Field                | Type    | Description                                                                                  |
-| -------------------- | ------- | -------------------------------------------------------------------------------------------- |
-| `version`            | string  | Version following the [SimVer](http://www.simver.org/) format                                |
-| `network`            | string  | Answer can be `mainnet`, `testnet`, `regtest`                                                |
-| `blockheight`        | integer | The block height we are synced at.                                                           |
-| `sync`               | float   | The synchronization progress as percentage (`0 < sync < 1`)                                  |
-| `descriptors`        | object  | Object with the name of the descriptor as key and the descriptor string as value             |
+| Field                | Type    | Description                                                                                        |
+| -------------------- | ------- | -------------------------------------------------------------------------------------------------- |
+| `version`            | string        | Version following the [SimVer](http://www.simver.org/) format                                |
+| `network`            | string        | Answer can be `mainnet`, `testnet`, `regtest`                                                |
+| `blockheight`        | integer       | The block height we are synced at.                                                           |
+| `sync`               | float         | The synchronization progress as percentage (`0 < sync < 1`)                                  |
+| `descriptors`        | object        | Object with the name of the descriptor as key and the descriptor string as value             |
+| `rescan_progress`    | float or null | Progress of an ongoing rescan as a percentage (between 0 and 1) if there is any              |
```

Comment:
It feels like it is a duplicate of `sync`, but maybe you are right to add it explicitly

Reply:
I think it makes sense to have both in the raw RPC interface. It could make sense to display it similarly to the sync progress in the GUI though?